### ISSUE TITLE
meanworks verify for mongodb problem 2

### ISSUE THREAD

**Issue by jsjqian** [azat-co/meanworks#3]
Because of a space in the folder name, an error is produced when running 'meanworks verify' for problem 2

**Comment by TylerMoeller** [comment]
Same here - fails on Windows but passes on OS X.

```
C:\meanworks>meanworks verify
MEANWorks is verifying that MongoDB is installed... Please stand by
Uh oh!  mongodb had a problem! {"killed":false,"code":1,"signal":null,"cmd":"C:\\Windows\\system32\\cmd.exe /s
 /c \"\"\"c:\\Program Files\\MongoDB\\Server\\3.2\\bin\\mongod.EXE\"\" --version\""}
'""c:\Program' is not recognized as an internal or external command,
operable program or batch file.
#########################################
###   YOUR SOLUTION IS NOT CORRECT!   ###
#########################################
```

The fix is to remove lines 22-25 from [index.js](https://github.com/azat-co/meanworks/blob/master/problems/02-mongodb/index.js):

``` javascript
    // Surrounds with double quotes if path contains a space
    if (mongod.indexOf(' ') != -1) {
      mongod = '"' + mongod + '"';
    }
```

I haven't tried that on Linux, but it works on Windows + OS X without these lines of code.

Another possible alternative is to skip if on Windows (and change `!=` to `!==` while we're at it): 

``` javascript
var os = require('os');
if (os.type() !== 'Windows_NT') {    
// Surrounds with double quotes on non-Windows OS if path contains a space
  if (mongod.indexOf(' ') !== -1) {
    mongod = '"' + mongod + '"';
  }
}
```

If you are happy with one of these solutions, I'm happy to submit a PR. Just notifying you with more details for now.

**Comment by jsjqian** [comment]
I implemented the first change and it worked perfectly. Thanks for the quick response and solution!

**Comment by azat-co** [comment]
@jsjqian @TylerMoeller i pushed the fix in 2.0.6: I had duplicate code doing the same thing. https://github.com/azat-co/meanworks/commit/dcea9e45f394cae77d14d3aec0a9d4860b7791c9
